app/templates/search/briefs.html

```diff
@@ -5,36 +5,18 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {% with %}
-    {% if current_lot %}
-      {% set items = [
-        {
-          "link": url_for('.index'),
-          "label": "Digital Marketplace"
-        },
-        {
-          "link": url_for('.{}'.format(view_name), framework_family=framework_family),
-          "label": "Supplier opportunities"
-        },
-        {
-          "label": current_lot.name
-        }
-      ] %}
-    {% else %}
-      {% set items = [
-        {
-          "link": url_for('.index'),
-          "label": "Digital Marketplace"
-        },
-        {
-          "link": url_for('.{}'.format(view_name), framework_family=framework_family),
-          "label": "Supplier opportunities"
-        }
-      ] %}
-    {% endif %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
-{% endblock %}
+{{ govukBreadcrumbs({
+  "items": [
+    {
+      "href": url_for('main.index'),
+      "text": "Digital Marketplace"
+    },
+    {
+      "text": "Supplier opportunities"
+    },
```

Comment:
This should have a link if there is a lot selected (i.e. it's not the final breadcrumb)

Reply:
Great spot, done 👍